assets/dev/js/editor/regions/responsive-bar/view.js

```diff
@@ -122,11 +123,14 @@ export default class View extends Marionette.ItemView {
 	}
 
 	onBreakpointSelected( e ) {
-		const currentDeviceMode = elementor.channels.deviceMode.request( 'currentMode' ),
-			selectedDeviceMode = e.target.value;
+		const selectedDeviceMode = e.target.value;
+
+		elementor.changeDeviceMode( selectedDeviceMode, false );
 
-		if ( currentDeviceMode !== selectedDeviceMode ) {
-			elementor.changeDeviceMode( selectedDeviceMode, false );
+		if ( 'widescreen' === selectedDeviceMode ) {
+			this.autoScale();
+		} else {
+			this.resetScale();
```

Comment:
Always do autoscale. Change the autoscale functionality to set width to 100% when the screen is not smaller than the preview